### DISCUSSION TITLE
Unregister Device?

### DISCUSSION THREAD

**Discussion by mawil583** [rxhanson/RectanglePro-Community#209] [Q&A] [answered]
I have Rectangle Pro registered on a work laptop for a company that I am now leaving. I know that I'm limited in how many devices I can have for a single purchase, so I would like to unregister my work laptop. But when I go into the settings, the closest thing I see is "Deactivate License." But I don't think that's what I want, because a license is what grants me access to register multiple devices, right? I'm looking for an "Unregister Device". How do I do this? Or is it as simple as uninstalling it from my machine?

Thanks

**Comment by rxhanson** [comment, marked as answer]
Deactivate License is what you're looking for, and now that you mention it, that is a confusing title. 

Deactivate license will remove that activation on that device, not kill your license key.

If you end up in a situation where you have not removed activations on devices ahead of time, and you run out of activations available, there is now a button in the prefs window when it has not been activated that allows you to remove an old activation.

**Comment by mawil583** [comment]
Thanks for the quick response!
